network-api/networkapi/utility/faker/streamfield_provider.py

```diff
@@ -159,7 +159,7 @@ def generate_video_field():
         # Fake embed url will lead to timeout error for Percy.
         # Solution: either we provide a valid embed url or use an empty string.
         # See details: https://github.com/mozilla/foundation.mozilla.org/issues/3833#issuecomment-562240677
-        'url': '',
+        'url': 'https://www.youtube.com/embed/83fk3RT8318',
```

Comment:
can we at least have a URL that is owned by us? I'd like to make sure we're not driving traffic to youtube just because as testing,

Reply:
Hi @Pomax! 

Since using the example.org URL would prevent the video component from rendering for testing, I took a look at the mozilla youtube channel and found [this](https://www.youtube.com/watch?v=COU5T-Wafa4) video on "how to choose strong passwords" from ~10 years ago. I was wondering if this would be a good candidate for the default URL? 

Tagging @KalobTaulien in case he has any input as well :) 

Thanks all!

Reply:
@Pomax the example YouTube video included here is under Firefox's YouTube account: https://www.youtube.com/watch?v=83fk3RT8318 so we should be good. Or we can find a Mozilla video if that's more preferred. As Daniel mentioned above, we need a real embed url for the video embed to show up properly so unfortunately example.com URLs won't work for this use case.